components/root_layout/root_layout.stories.js

```diff
@@ -8,14 +8,15 @@ import {
 import DtRootLayout from './root_layout.vue';
 
 import DtRootLayoutDefaultTemplate from './root_layout_default.story.vue';
+import DtRootLayoutStickyTemplate from './root_layout_sticky.story.vue';
 
 // Default Prop Values
 export const argsData = {
   sidebarPosition: 'left',
-  header: '<div class="d-bgc-purple-200 d-h64 d-h100p">Header</div>',
-  footer: '<div class="d-bgc-gold-200 d-h64 d-h100p">Footer</div>',
+  header: '<div class="d-bgc-purple-200 d-h64">Header</div>',
+  footer: '<div class="d-bgc-gold-200 d-h64">Footer</div>',
   sidebar:
-    '<div class="d-bgc-black-200 d-h100p d-w264"><div>Sidebar item 1</div><div>Sidebar item 2</div><div>Sidebar item 3</div></div>',
+    '<div class="d-bgc-black-200 d-wmn264 d-h100p"><div>Sidebar item 1</div><div>Sidebar item 2</div><div>Sidebar item 3</div></div>',
```

Comment:
`d-h100p` shouldn't be necessary here anymore, right?

Reply:
it's basically just for the example. The sidebar itself takes up the full height whether you have this or not, but this element within the sidebar will not take up the full height unless you tell it to. Basically just using it to set the whole sidebar to a color.